device-types/Dell/PowerEdge_R620xl_solidfire.yml

```diff
@@ -0,0 +1,39 @@
+---
+manufacturer: Dell
+model: SolidFire OEMR R620 XL
+slug: dell_solidfire_omer_r620xl
+u_height: 1
+is_full_depth: true
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: USB
+    type: usb-a
```

Comment:
We are no longer including USB ports as "console" unless they are specifically console ports